.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,15 +14,21 @@ _Note: Before submitting a pull request, please open an issue for discussion if
 Closes #ISSUE_NUMBER
 
 <!--
-For example, if your pull requests resolves issues 1000 and 2000 write:
+For example, if your pull requests resolves multiple issues, such as 1000 and 2000 write:
 * Closes #1000
 * Closes #2000
 -->
 
-## Checklist
+## Checklists
+
+#### Both external and internal contributors
 - [ ] I have followed the [`CONTRIBUTING` Guide](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/CONTRIBUTING.md).
 - [ ] I have commented my code, particularly in hard-to-understand areas
-- [ ] I have successfully run the E2E tests, and have included the outcomes in the PR comments
-- [ ] I have added unit and integration tests relevant to my change as needed
-- [ ] New and existing unit tests pass locally with my changes (suggestion: use the `tests/local_check.sh` script)
-- [ ] I have updated any relevant documentation to reflect my changes
+- [ ] I have added unit and/or integration tests relevant to my change as needed
+- [ ] I have updated any relevant documentation to reflect my changes, if applicable
+
+#### Internal contributors only
+- [ ] I already checked locally that all unit tests and linting are passing (use the `tests/local_check.sh` script)
```

Comment:
External contributors should locally check unit tests pass + run unit testing on their own DB. I don't think we should have different lists for internal vs external - we should just merge this into one list for everyone